Source/WTF/Configurations/module.modulemap

```diff
@@ -0,0 +1,62 @@
+module wtf {
+    // AutodrainedPool must never be parsed in an objective-C
+    // context.
+    explicit module AutodrainedPool {
+        requires cplusplus11, !objc
+        header "AutodrainedPool.h"
+        export *
+    }
+    // This header depends on surrounding preprocessor context.
+    // Works only on some platforms.
+    explicit module MetadataSPI {
+        requires cplusplus11
+        textual header "spi/mac/MetadataSPI.h"
```

Comment:
This can't be a `textual` header because it has declarations, and more than one includer